### PR TITLE
 bugfix/GRW-772/Add voucher Code

### PR DESCRIPTION
**Pull request by dendoesit**
## What?

Fix the add voucher code button on the offer page. Reported in the quote cart testing session. 

## Why?

I believe this breaks after the new input-fields PR and I think it is easier to roll forward than roll back. There was some code scouting to be done in the file but I did not want to block the release. 

## Screenshots / recordings 
Now if you type something the button becomes clickable. 
image

### [Review app]()

https://web-onboardi-bugfix-grw-kykdxa.herokuapp.com/se-en/new-member/debugger

### CODE REVIEW

**Review by robinandeer — COMMENTED** [review]
Would this be the same problem as in #798? Then we just need to pass a "name" attribute and it will work, no?

also => do you wanna make the changes for the new quote cart offer page as well? 🙏

File: src/client/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx [hunk 1]
```diff
@@ -145,7 +146,7 @@ export const DiscountCodeModal: React.FC<Props> = ({
               })
           }
         >
-          {({ touched, errors, values }) => (
+          {({ touched, errors }) => (
             <Form>
               <InputField
                 label={textKeys.SIDEBAR_ADD_DISCOUNT_CELL_LABEL()}
```
Comment (guilhermespopolin) [inline src/client/pages/OfferNew/Introduction/Sidebar/DiscountCodeModal.tsx:152]:
```suggestion
          {({ getFieldProps, values, touched, errors, isSubmitting }) => (
            <Form>
              <InputField
                label={textKeys.SIDEBAR_ADD_DISCOUNT_CELL_LABEL()}
                placeholder={textKeys.SIDEBAR_ADD_DISCOUNT_CELL_PLACEHOLDER()}
                  type="text"
                  touched={touched.code}
                  errors={errors.code ? textKeys[errors.code]() : ''}
                  {...getFieldProps('code')}
                />
                  <Footer>
                    <Button
                      type="submit"
                      fullWidth
                      disabled={!values.code.trim() || isSubmitting}
                    >
                      {textKeys.SIDEBAR_ADD_DISCOUNT_BUTTON()}
                    </Button>
                  </Footer>
              </Form>
            )}}
```

**Review by guilhermespopolin — CHANGES_REQUESTED** [review]
As @robinandeer has mentioned, it would be great if you also change `pages/Offer/Introduction/Sidebar/CampaignCodeModal.tsx` file.

**Review by guilhermespopolin — APPROVED** [review]
👍🏻

**Pull request merged**